src/encoded/schemas/ivp_a1.json

```diff
@@ -0,0 +1,396 @@
+      "title": "9a. Other (specify subjects race):",
+      "description":"(Any text or numbers with the exception of single quotes ('), double quotes (\"), ampersands (&), and percentage signs (%).)",
+      "type": "string"
+    },
+    "racesec": {
+      "title": "10. What additional race does subject report?",
+      "description": "If '50 Other (specify)', CONTINUE TO 10a",
+      "type": "string",
+      "enum": [
+        "1 White",
+        "2 Black or African American",
+        "3 American Indian or Alaska Native",
+        "4 Native Hawaiian or other Pacific Islander",
+        "5 Asian",
+        "50 Other (specify)",
+        "88 None Reported",
+        "99 Unknown"
+      ]
+    },
+    "racesecx": {
+      "title": "10a. Other (specify subject's additional race):",
+      "description":"(Any text or numbers with the exception of single quotes ('), double quotes (\"), ampersands (&), and percentage signs (%).)",
+      "type": "string"
+    },
+    "raceter": {
+      "title": "11. What additional race, beyond what was indicated above in questions 9 and 10, does subject report?",
+      "description": "If '50 Other (specify)',CONTINUE TO 11a",
+      "type": "string",
+      "enum": [
+        "1 White",
+        "2 Black or African American",
+        "3 American Indian or Alaska Native",
+        "4 Native Hawaiian or other Pacific Islander",
+        "5 Asian",
+        "50 Other (specify)",
+        "88 None Reported",
+        "99 Unknown"
+      ]
+    },
+    "raceterx": {
+      "title": "11a. Other (specify subject's additonal race beyond questions 9 and 10):",
+      "description":"(Any text or numbers with the exception of single quotes ('), double quotes (\"), ampersands (&), and percentage signs (%).)",
+      "type": "string"
+    },
+    "primlang": {
+      "title": "12. Subject's primary language:",
+      "description": "If Other(SPECIFY),CONTINUE TO 12a",
+      "type": "string",
+      "enum": [
+        "1 English",
+        "2 Spanish",
+        "3 Mandarin",
+        "4 Cantonese",
+        "5 Russian",
+        "6 Japanese",
+        "8 Other primary language (specify)",
+        "9 Unknown"
+      ]
+    },
+    "primlanx": {
+      "title": "12a. Other (specify subject's primary language):",
+      "description":"(Any text or numbers with the exception of single quotes ('), double quotes (\"), ampersands (&), and percentage signs (%).)",
+      "type": "string"
+    },
+    "educ": {
+      "title": "13. Subject's years of education-(Use the codes here to report the level achieved; if an attempted level is not completed, enter the number of years completed.)",
+      "description": "12 = High school or GED,16 = Bachelor's degree,18 = Master's degree,20 = Doctorate,99 = Unknown",
+      "type": "string",
+      "enum": [
+          "1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","99"
```

Comment:
The other years are addtional years after docterate (e.g Post-doc, residency, fellowship.)

Reply:
1. String or integer?(both work with enum),which one is more convenient for future data calculation? 
2. What will happen if the additional year equal to 12,16,18,20?

Reply:
i think you can make integer easier for future

Reply:
![image](https://user-images.githubusercontent.com/38848214/83100708-296fcf80-a076-11ea-84ff-83c3c8eb00ea.png)
Finally found this one.

Reply:
Fixed small bugs. ![image](https://user-images.githubusercontent.com/38848214/83102599-76ee3b80-a07a-11ea-84a6-c2d1fe604081.png)
![image](https://user-images.githubusercontent.com/38848214/83102624-87061b00-a07a-11ea-937a-b9b8f19ce3ed.png)